src/api/app/helpers/webui/staging/workflow_helper.rb

```diff
@@ -43,20 +43,43 @@ def progress(staging_project)
     end
   end
 
+  def reviewers_gravatar(request)
+    missing_reviews = request[:missing_reviews]
+    image_tags = []
+    missing_reviews.each do |review|
+      case review[:review_type]
+      when 'by_group'
+        group = Group.find_by_title(review[:by])
+        image_tags << webui2_user_or_group_image_tag(user_or_group: group, title: review[:by], size: 20)
+      when 'by_user'
+        user = User.find_by_login(review[:by])
+        image_tags << webui2_user_or_group_image_tag(user_or_group: user, title: review[:by], size: 20)
+      end
```

Comment:
there is no else - as such you would miss by_project and by_package

Reply:
We could define a default by_project and a default by_package icon that we show in such cases. But this can be done in a follow up PR.

Reply:
@coolo 

> there is no else - as such you would miss by_project and by_package

@Ana06 and I will do this in an follow up PR